client/api/filesystem/v1beta2/api.proto

```diff
@@ -5,21 +5,21 @@ package v1beta2;
 option go_package = "github.com/kubernetes-csi/csi-proxy/client/api/filesystem/v1beta2";
 
 service Filesystem {
-    // PathExists checks if the requested path exists in the host's filesystem
+    // PathExists checks if the requested path exists in the host filesystem.
     rpc PathExists(PathExistsRequest) returns (PathExistsResponse) {}
 
-    // Mkdir creates a directory at the requested path in the host's filesystem
+    // Mkdir creates a directory at the requested path in the host filesystem
     rpc Mkdir(MkdirRequest) returns (MkdirResponse) {}
 
-    // Rmdir removes the directory at the requested path in the host's filesystem.
-    // This may be used for unlinking a symlink created through LinkPath
+    // Rmdir removes the directory at the requested path in the host filesystem.
+    // This may be used for unlinking a symlink created through LinkPath.
     rpc Rmdir(RmdirRequest) returns (RmdirResponse) {}
 
     // LinkPath creates a local directory symbolic link between a source path
-    // and target path in the host's filesystem
+    // and target path in the host's filesystem.
```

Comment:
One feedback about name is CreateSymlink and IsSymlink

Reply:
Added this in the description of #133 and also in this PR